conf/airframes/examples/bebop.xml

```diff
@@ -10,6 +10,17 @@
       <module name="udp"/>
     </target>
 
+    <define name="MT9F002_OUTPUT_HEIGHT" value="640" />
```

Comment:
@flixr where is a good place to put this, the module config maybe. I guess we want it to be added to the doxygen as well, not just the example airframe.